src/pages/api/admin/validate.ts

```
@@ -0,0 +1,33 @@
+import { PrismaClient } from '@prisma/client';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+const prisma = new PrismaClient();
+
+const adminValidate = async (req: NextApiRequest, res: NextApiResponse) => {
+  const userEmail = req.query.email as string;
+  try {
+    const user = await prisma.user.findUnique({
+      where: { email: userEmail }
+    });
+    if (!user) {
+      return res.status(404).json({
+        message: 'User not found'
+      });
+    }
+    const account = await prisma.account.findMany({
+      where: {
+        userId: user.id,
+        type: 'admin'
+      }
+    });
+    if (account.length > 0) {
```

What if multiple accounts are returned - we should throw an error in this case